src/arguments/api_test.py

```diff
@@ -42,7 +42,39 @@ def fetch_input_url(cls):
             "request_url" : request_url,
             "request_headers" : request_headers,
         }
-
+    @classmethod
+    def read_data_from_file(cls):
+        filename = input("Enter a filename (response_data.json)")
+        if filename == '':
+            print("filename empty, so default file (response_data.json) is used ")
+        with open(filename, "r") as reader:
+            file_content = reader.read()
+            try:
+                json_data = json.loads(file_content)
+                data = json_data.get("data")
+                return data
+                # Make sure the data is not None and send
+            except json.JSONDecodeError:
+                 print("Unable to parse the file, Please try again")
+                 cls.read_data_from_file()
+    @classmethod
+    def fetch_payload_data(cls):
+        store = int(input("Please choose the below options? (1/2/3)"))
```

Comment:
Can we add a option here ?
Do you want to send payload Y/N
Then we can display option like send a payload from terminal or file

Reply:
then there would be multiple conditions 
``` 
   if yes :
       enter 1 to send from terminal
       enter 2 to read from terminal
       any other number #then again we have to repeat from starting line, if you want to send payload
   elif no 
       return 
   else
          recur the function

Reply:
Do we also send  bearer token in this payload ?

Reply:
yeah if the user sends it